packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -98,9 +98,11 @@
 
   &__dialog-heading {
     margin: 0;
-    font-weight: 700;
 
     @include bpk-text-base;
```

Comment:
@jamesf3rguson should the `bpk-text-bold` mixin always set the font-weight to "700"? What should it do in the iOS theme? (Option: not set anything at all)